bbc/programmes-frontend
# Lazysizes library added

**To test lazyload behaviour:**
Just scroll the page and see in dev tools the requests for network/images

**To test responsive sizes:**
With different zooms reload the page. Check that for 'big-zoom out', images are requested with lower resolution than for 'big-zoom in'.
For instance:
With big zoom-out: https://ichef.bbci.co.uk/images/ic/320x180/p057qg1r.jpg
With big zoom-in: https://ichef.bbci.co.uk/images/ic/480x270/p057qg1r.jpg
